src/email_requests/views.py

```diff
@@ -59,3 +59,14 @@ def post(self, request, *args, **kwargs):
         new_request.save()
 
         return Response(RequestSerializer(new_request).data)
+
+
+class ListOwnRequestsView(generics.ListAPIView):
+    serializer_class = RequestSerializer
+
+    def get_queryset(self):
+        mentor = get_object_or_404(Mentor, profile__user=self.request.user)
+        return Request.objects.filter(mentor=mentor).order_by('date_created').reverse()
```

Comment:
I might have missed this before, but do we have a concept of active vs completed requests?

Reply:
No, everything about a request between mentor and mentee is handled outside of the platform once the email is sent